web/packages/teleterm/src/ui/ClusterLogout/useClusterLogout.ts

```diff
@@ -30,6 +30,9 @@ export function useClusterLogout({
   const ctx = useAppContext();
   const [{ status, statusText }, removeCluster] = useAsync(async () => {
     await ctx.clustersService.logout(clusterUri);
+    await ctx.mainProcessClient.maybeRemoveAppUpdatesManagingCluster(
+      clusterUri
+    );
```

Comment:
Does this line in combination with the following one mean that when the user logs out of a cluster, the whole logout procedure won't be finished until the app finishes a check for updates?

https://github.com/gravitational/teleport/blob/511a37de71f197cec5f421836e4c28e545f79491/web/packages/teleterm/src/services/appUpdater/appUpdater.ts#L270-L277

If that's the case, I feel like logging out of a cluster shouldn't be allowed to wait for an update check of shouldn't be forced to fail if something goes wrong with auto-updates. Perhaps it should be done in a background instead and possible errors could be logged or shown as a notification? I assume the user won't care about the error until they actually open the update modal.

Reply:
Great catch, we shouldn't wait for it.